src/app/shared/messages/auth.messages.ts

```diff
@@ -0,0 +1,7 @@
+// TODO: Add message codes [Error] [Success] [Info] etc
+
+export const LoginSuccessMsg = "Successfully Logged In!";
```

Comment:
Constants must be all capitals.

Reply:
followed the same conventions at most places. Will have to create a separate issue if changes are required.